packages/lib-subject-viewers/.storybook/preview.js

```diff
@@ -0,0 +1,61 @@
+import { INITIAL_VIEWPORTS } from '@storybook/addon-viewport'
+import i18n from '../src/translations/i18n'
+import zooTheme from '@zooniverse/grommet-theme'
+import { Grommet } from 'grommet'
```

Comment:
These lines are copy-pasted from another library. Please refactor to match the needs of lib-subject-viewers. `i18n` should wait for actual content, and you're missing grommet + styling packages from a `peerDependencies` section in package.json.